pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -490,6 +508,14 @@ type FrameworkHandle interface {
 	ClientSet() clientset.Interface
 
 	SharedInformerFactory() informers.SharedInformerFactory
+
+	// PreemptHandle returns a preemption handle which can help on running PostFilter plugins.
+	PreemptHandle() PreemptHandle
```

Comment:
I don't the name `PreemptHanlde` fits the long-term purpose anymore. Any name suggestions? For example, `RuntimeHelper` to denote it exposes some runtime functions?

Reply:
I don't mind keeping the name PreemptHandle, RuntimeHelper is not specific enough, we already have FrameworkHandle which is very generic.

One thing we can do is completely remove PeemptHandle and just unroll the interface in FrameworkHandle.

Reply:
That's fine, but let's not forget how much the interface is growing. Maybe we can preserve the PodNominator and PluginsRunner interfaces.

Reply:
yes, my suggestion is just to unroll PreemptHandle only, but not the interfaces that it encapsulates.

Reply:
SG.

Reply:
lets do that in a separate PR and focus this PR on PostFilter only, so lets remove this line from this PR.

Reply:
Added a comment here.